pkg/istio-agent/sds-agent.go

```diff
@@ -353,13 +357,20 @@ func newSecretCache(serverOptions sds.Options) (workloadSecretCache *cache.Secre
 				log.Info("Using citadel CA for SDS")
 				serverOptions.CAEndpoint = "istio-citadel.istio-system:8060"
```

Comment:
This is no longer correct, should be istiod.istio-system.svc:15012 I think ( or istio-pilot... if we don't create the new svc name )

Reply:
Revised.